_source/_posts/2019-09-30-java-quarkus-oidc.md

```diff
@@ -508,3 +505,7 @@ Here are some related blog posts to learn more about Java and authentication:
 - [Build a Simple CRUD App with Spring Boot and Vue.js](/blog/2018/11/20/build-crud-spring-and-vue)
 
 If you have any questions about this post, please add a comment below. For more awesome content, follow  [@oktadev](https://twitter.com/oktadev) on Twitter, like us [on Facebook](https://www.facebook.com/oktadevelopers/), or subscribe to [our YouTube channel](https://www.youtube.com/c/oktadev).
+
+### Changelog
+
+* January 27, 2021: Update Quarkus to version 1.11.1.Final
```

Comment:
Can you please change to follow syntax like the following (no header for the changelog, the month should be three characters, links to both PRs).

```
<a name="changelog"></a>
**Changelog**:

* Jan 26, 2021: Updated post to use Spring Boot 2.4.2 and Spring Data 2020.0. See the code changes in the [example on GitHub](https://github.com/oktadeveloper/okta-spring-data-jpa-example/pull/4). Changes to this post can be viewed in [oktadeveloper/okta-blog#536](https://github.com/oktadeveloper/okta-blog/pull/536).
```

Reply:
Easy enough!